var/spack/repos/builtin/packages/singularity/package.py

```diff
@@ -6,8 +6,6 @@
 from spack import *
 
 import llnl.util.tty as tty
-import os
-import shutil
 
 
 class Singularity(MakefilePackage):
```

Comment:
@ArangoGutierrez @alalazo will want to review the changes to this package.

Reply:
@alalazo is listed as a reviewer.  Can't seem to tag @ArangoGutierrez, but presumably you and I mentioning him will get his attention.

I was waiting for the dust to settle, but @vsoch and @odoublewen are probably interested too.